podcasts.xcodeproj/xcshareddata/xcschemes/Pocket Casts Prototype Build.xcscheme

```diff
@@ -120,7 +103,7 @@
       buildConfiguration = "Debug">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Prototype"
       revealArchiveInOrganizer = "YES">
    </ArchiveAction>
 </Scheme>
```

Comment:
This scheme looks like and edit of an existing in this diff because of how textual diffs work, but once #1767 lands it'll be a new one.